introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello, my name is <insert name here> and I like to eat <insert food here>.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?